src/screens/FeedScreen/RenderList.js

```diff
@@ -84,6 +89,8 @@ const RenderListFeed = (props) => {
     });
   };
 
+  const {followUnfollow} = usePostHook();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new hook `usePostHook` has been used to get the `followUnfollow` function. Make sure that this function is used correctly in the code.
```diff
+   const {followUnfollow} = usePostHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->